docs/products/opensearch/reference/plugins.rst

```diff
@@ -1,7 +1,7 @@
 Plugins available with Aiven for OpenSearch
 ===========================================
 
-Aiven for OpenSearch includes by default a standard a set of plugins. We are also working on adding a further set of new OpenSearch plugins. The exact plugins are listed in the sections further down this page.
+Aiven for OpenSearch includes a standard a set of plugins. In addition to the plugins that were previously available in Aiven for Elasticsearch, the standard set of plugins also includes some that are new for OpenSearch.
```

Comment:
a standard set (extra 'a')

Reply:
True, well spotted - that's now fixed ;)